spec/tapioca/cli/gem_spec.rb

```diff
@@ -2022,6 +2022,24 @@ class << self
           OUT
         end
       end
+
+      it "generates correct RBIs for a Rails engine" do
+        @project.require_real_gem("rails")
+        @project.require_real_gem("turbo-rails", "1.3.2")
+        @project.bundle_install
+
+        response = @project.tapioca("gem")
+
+        assert_includes(response.out, "Compiled turbo-rails")
+
+        turbo_streams_rbi = @project.read("sorbet/rbi/gems/turbo-rails@1.3.2.rbi")
+
+        # The Turbo::Streams constant is never defined on its own, only as a namespace for child constants.
+        # Because of this, tapioca cannot create an annotation for it using traditional auto-loading methods
+        # By leveraging Zeitwerk auto-loading, tapioca can discover this constant and correctly create an
+        # annotation for it.
+        assert_includes(turbo_streams_rbi, "module Turbo::Streams; end")
+      end
```

Comment:
@paracycle Let me know if this test looks reasonable to you. I've confirmed that undo-ing our changes breaks the test.